docker/testdata/test_grass_session.py

```diff
@@ -1,18 +1,16 @@
-# Import GRASS Python bindings
-# https://github.com/zarch/grass-session
-# pip install grass-session
+# Import GRASS GIS Python bindings (requires 8-4+) and test r.in.pdal
```

Comment:
What does 8-4+ mean? Was it meant to be 8.4+?

Reply:
Yes